chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestSuite.java

```diff
@@ -0,0 +1,52 @@
+package com.griddynamics.jagger.user.test.configurations;
+
+import java.util.List;
+
+/**
+ * Defines the test suite consisting of several {@link JTestGroup}.
+ */
+public class JTestSuite {
+
+    private List<JTestGroup> testGroups;
+
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    private JTestSuite(Builder builder) {
+        this.testGroups = builder.testGroups;
+    }
+
+
+    public static class Builder {
+        private List<JTestGroup> testGroups;
+
+        private Builder() {
+        }
+
+        /**
+         * Sets the {@code testGroups} for test suite.
```

Comment:
+ test groups will be executed in the order from beginning to the end of the list